ush/forecast_postdet.sh

```diff
@@ -876,15 +876,13 @@ MOM6_postdet() {
   elif [[ "${CDUMP}" =~ "gdas" ]]; then
     # Link output files for CDUMP = gdas
 
-    # MOM6 does not write out the first forecast hour, so start at FHOUT
-    local fhr="${FHOUT}"
-    while [[ "${fhr}" -le "${FHMAX}" ]]; do
+    # Save MOM6 backgrounds
+    local fhr="${FHMIN}"
+    for fhr in ${OUTPUT_FH}; do
```

Comment:
Technically, `OUTPUT_FH` is only valid for atmosphere output.  That was the reason for the while loop.  If this works for you right now, then its fine for now.
MOM6 output frequency is controlled differently.